stacks.yml

```diff
@@ -30,10 +31,13 @@ data_storage:
   - documents_s3
   - logs_s3
   - elasticsearch
+  - monitoring
```

Comment:
I don't think this should be here. The specific stacks that require it should depend on it.